### DISCUSSION TITLE
varied aspect ratio

### DISCUSSION THREAD

**Discussion by amir-danesh** [nanxiaobei/antd-img-crop#196] [Q&A]
How can I make the aspect attribute work so that the user can change the ratio of the crop?

**Comment by nanxiaobei** [comment]
`aspect={9/16}`, like this

**Comment by amir-danesh** [comment]
@nanxiaobei no i mean like resize. i want the user to adjust the size by themselves. you had this feature in version 2.5.0 I guess

**Reply by nanxiaobei** [reply]
sorry does not support resize the crop manually now.
